docs/MANUAL.md

```diff
@@ -657,11 +657,11 @@ Download with `PORT` but use 192.168.0.10 as our IP address to use:
 
 Get a web page from a server using a specified port for the interface:
 
-    curl --interface eth0:1 http://www.netscape.com/
+    curl --interface eth0:1 http://www.example.com/
```

Comment:
eth0 is a bit too ungenric IMO

Reply:
What would you suggest instead? `if0`?

Reply:
I think `eth0` is good there because it is known and recognizable as a network interface name.